.azure-pipelines/tests.yml

```diff
@@ -0,0 +1,72 @@
+ernal variables:
```

Comment:
Looks like you accidentally deleted the beginning of this comment.

Reply:
🤦‍♂️